setup.py

```diff
@@ -17,7 +17,7 @@
 setup(
     name=base_name,
     version='1.0',
-    author=u'Nebra Ltd.',
+    author='Nebra Ltd.',
```

Comment:
Any particular reason this was removed? Seems like its a Python 2 compatibility thing and not relevant either way in Python 3.

Reply:
Python 2 requires the author string to be in unicode? I found it odd that it was being converted from a byte string into a unicode one here, if this causes issues with python2 I will revert it immediately